src/crewai/flow/flow.py

```diff
@@ -465,7 +469,9 @@ def __init__(
 
         # Initialize state with initial values
         self._state = self._create_initial_state()
-
+        if os.getenv("CREWAI_TRACING_ENABLED", "false").lower() == "true":
```

Comment:
We currently have those env vars related to telemetry, can we reuse any of them?

```
CREWAI_DISABLE_TELEMETRY
CREWAI_DISABLE_TRACKING
```

Just trying to avoid adding unnecessary env-vars